<?xml version="1.0" ?>
<launch>
 <group ns="robot0">
  <node pkg="victim_detection" type="find_victim_pose" name="victim_position2" output="screen">
    <param name="robot_namespace" value="robot0"/>
    <param name="number_of_rays" type="int" value="181"/>
    <param name="width_of_image" type="int" value="320"/>
    <param name="angle_range_camera" value="59.989"/>
    <!--<param name="angle-range-laser" type ="int" value="2*((1.570796*180)/PI)"/> -->
    <param name="angle_range_laser" type ="int" value="180"/>
  </node>
 </group>
</launch>
